cmd/inventory.go

```diff
@@ -124,6 +124,7 @@ func (i *inventoryCommand) inventoryAgent() error {
 	fmt.Printf("  Configuration Management Classes:\n\n")
 	longest := util.LongestString(inventory.Classes, 40)
 	format := fmt.Sprintf("    %%-%ds %%-%ds\n", longest, longest)
+	sort.Strings(inventory.Classes)
```

Comment:
Lets sort here instead and we solve it for all use cases

https://github.com/choria-io/go-choria/blob/97b23c3eab4ce4810fe35f66d284f2d3141f1068/filter/classes/classes.go#L94

no issue is fine

Reply:
Makes sense, but now the client blindly displays the list in the order provided by the server.  That might be surprising in an environment with different versions of choria, but I am perfectly fine with this.

I adjusted the PR accordingly.